pkg/leaderelection/leaderelection.go

```diff
@@ -30,18 +30,24 @@ func IsLeader() (bool, error) {
 }
 
 func getLeader(electorPath string) (string, error) {
-	resp, err := electorRequestWithRetry(electorPath, 3)
+	const numRetries = 3
+
+	resp, err := electorRequestWithRetry(electorPath, numRetries)
 	if err != nil {
 		return "", err
 	}
-	defer resp.Body.Close()
+
+	defer func(Body io.ReadCloser) {
+		_ = Body.Close()
+	}(resp.Body)
```

Comment:
Hvorfor vil du skjule feilen man får fra `Body.Close()`? Forsvinner ikke hele poenget med denne fancy `defer` da?

Reply:
Her brukes defer av en funksjon for å være eksplisitt på at man har valgt å ignorere `err` med vilje, i praksis er dette samme logikk som vi allerede har, bare at vi sier: "vi gjør dette med vilje". Har egentlig ikke tatt stilling til om man faktisk burde håndtere err her eller ikke

Reply:
I mitt hode så ignorerer man alltid de `err` fra `defer`, så i mine øyne blir det eksplisitt eksplisitt :p Blir litt smør på flesk.

Reply:
Yeah, ser den, mulig å disable `errcheck`, men synes den er ganske fin mange ganger